README.md

```diff
@@ -57,10 +57,16 @@ The following token needs to be set in the Github repo for the `prerelease` and
 
 - `secrets.NPM_TOKEN` (need this to publish on NPM)
 
+To properly release remember to follow our conventional commit approach as stated in [contributing](https://github.com/lokalise/npm-package-template/blob/main/CONTRIBUTING.md). 
+
 ## License
 
 This project is APACHE, VERSION 2.0 licensed, see LICENSE.md for details.
 
 ## Other Resources
 
 This template represents the culmination of Lokalise technical recommendations as documented in the [Frontend Radar](https://lokalise.atlassian.net/l/cp/Bqkz2hC5).
+
+## Support Us
+
+`lokalise/npm-package-template` was created by Lokalise Engineering Team. Support our work by keeping this line in your README.
```

Comment:
The reason I'm adding this line is to be able to search all repos created through this template with a global search from GitHub (https://github.com/search/advanced).
That way we could do something like: https://github.com/search?l=&q=lokalise/npm-package-template+in%3AREADME